samples/drivers/video/capture/sample.yaml

```diff
@@ -9,20 +9,25 @@ tests:
     extra_args:
       - platform:mimxrt1064_evk:SHIELD="dvp_fpc24_mt9m114;rk043fn66hs_ctg"
       - platform:mimxrt1170_evk/mimxrt1176/cm7:SHIELD="nxp_btb44_ov5640;rk055hdmipi4ma0"
+      - platform:mimxrt1170_evk@B/mimxrt1176/cm7:SHIELD="nxp_btb44_ov5640;rk055hdmipi4ma0"
```

Comment:
this still depends on the PR #79263

Reply:
Resolved as PR https://github.com/zephyrproject-rtos/zephyr/pull/79263 is now merged